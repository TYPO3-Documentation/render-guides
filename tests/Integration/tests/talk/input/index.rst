
..  _start:

==============
talk.typo3.org
==============

..  typo3:talk::
