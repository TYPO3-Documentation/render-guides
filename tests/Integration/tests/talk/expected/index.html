<!-- content start -->
                <section class="section" id="talk-typo3-org" data-rst-anchor="start">
            <a id="start"></a>
            <h1>talk.typo3.org<a class="headerlink" href="#talk-typo3-org" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline"><i class="fa-solid fa-paragraph"></i></a></h1>
            <div id="discourse-comments" class="mt-5"></div>

<script>
    DiscourseEmbed = {
        discourseUrl: 'https://talk.typo3.org/',
        discourseEmbedUrl: '[Topic on docs.typo3.org](https://docs.typo3.org/permalink/changelog:start)'
    };

    (function () {
        var d = document.createElement('script');
        d.type = 'text/javascript';
        d.async = true;
        d.src = DiscourseEmbed.discourseUrl + 'javascripts/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(d);
    })();
</script>

    </section>
        <!-- content end -->
